### PR TITLE
highlight menu and breadcrumb

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2159
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/360
https://github.com/open-orchestra/open-orchestra/pull/1165

### CODE REVIEW

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/OrchestraRouter.js [hunk 1]
```diff
@@ -43,13 +44,45 @@ class OrchestraRouter extends Backbone.Router
     }
 
     /**
+     * @returns {Array}
+     * @private
+     */
+    getNavigationHighlight() {
+        return null;
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/OrchestraRouter.js]: return empty array

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/OrchestraRouter.js [hunk 2]
```diff
@@ -43,13 +44,45 @@ class OrchestraRouter extends Backbone.Router
...
+    }
+
+    /**
+     * @returns {Array}
+     * @private
+     */
+    getBreadcrumbHighlight() {
+        return null;
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/OrchestraRouter.js]: return empty array

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Site/SiteRouter.js [hunk 3]
```diff
@@ -77,12 +90,13 @@ class SiteRouter extends OrchestraRouter
     /**
      * New site
      */
-    newSite() {
+    newSite(inPlatformContext) {
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Site/SiteRouter.js]: an admin site can create a new site ?

**Comment by itkg-nanne** [comment]
@alavieille update done

**Comment by itkg-nanne** [comment]
update done @alavieille , @ngilain

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Application.js [hunk 4]
```diff
@@ -11,6 +12,7 @@ import TrashItemRouter         from './Router/TrashItem/TrashItemRouter'
 
 import HeaderView              from './View/Header/HeaderView'
 import ErrorView               from './View/Error/ErrorView'
+import NavigationManager       from '../Service/CourseManager'
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Application.js]: @itkg-nanne : I think there is a fail somewhere ^^"

File: BackofficeBundle/Resources/public/template/Menu/Contribution/trashcan._tpl.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <li>
-    <a href="#<%= Backbone.history.generateUrl('listTrashItem') %>">
+    <a href="#<%= Backbone.history.generateUrl('listTrashItem') %>" id="course-trashcan">
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/template/Menu/Contribution/trashcan._tpl.html]: update this part

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Block/BlockRouter.js [hunk 6]
```diff
@@ -33,16 +33,28 @@ class BlockRouter extends AbstractBlockRouter
...
             }
         ]
     }
 
     /**
+     * @inheritdoc
+     */
+    getMenuHighlight() {
+        return {
+            newBlockListComponent : 'navigation-node',
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Block/BlockRouter.js]: in all router, the menu highlight is the same for all route. so maybe return only a string

**Comment by itkg-nanne** [comment]
update done @alavieille , @ngilain

**Pull request merged**